engine/battle/core.asm

```diff
@@ -26,7 +26,7 @@ DoBattle: ; 3c000
 	and a
 	jr z, .not_linked
 
-	ld a, [hLinkPlayerNumber]
+	ld a, [hSerialConnectionStatus]
 	cp $2
```

Comment:
Some hSerialConnectionStatus constants (like USING_INTERNAL_CLOCK) are missing in code, more specifically in engine/battle/core.asm, home/init.asm, mobile/mobile_40.asm, and a few in engine/link.asm. Pointing out in case you haven't noticed, because you may already be aware.

Reply:
Thanks. Even with that the link code has much left to do (e.g. the `wPlayerLinkAction` and `wOtherPlayerLinkMode` values, the many `wXXXX` labels, hardcoded sizes like `$1b9`), but I don't have the knowledge of GB serial hardware to do it.